src/main/java/hw6/pages/JdiMetalsColorsPage.java

```diff
@@ -0,0 +1,68 @@
+package hw6.pages;
+
+import com.epam.jdi.light.elements.composite.WebPage;
+import com.epam.jdi.light.elements.pageobjects.annotations.FindBy;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.Css;
+import hw6.DataFromJsonForMetalsColorPage;
+import hw6.entities.Summary;
+import hw6.forms.SummaryForm;
+import org.openqa.selenium.By;
+
+import java.util.List;
+
+public class JdiMetalsColorsPage extends WebPage {
+
+    @Css(".main-content")
+    private SectionWhereUserChooseElements mainSection;
+
+    private SummaryForm summary;
+
+    @FindBy(xpath = "//div[@name='log-sidebar']")
+    private LogSection log;
+
+    public void setSummary(Summary summaryFields) {
+        summary.fill(summaryFields);
+    }
+
+    public void setVegetables(List<String> vegValues) {
+        mainSection.getVegetables().find(By.tagName("button")).click();
```

Comment:
This is not the best way to use findBy in PO methods. 
You can create your own Page Element or override existing one in order to change it behaviour.

Anyway, this method should not be in Page class, Form required.